services/hanke-service/src/main/kotlin/fi/hel/haitaton/hanke/HankeMapper.kt

```diff
@@ -0,0 +1,64 @@
+package fi.hel.haitaton.hanke
+
+import fi.hel.haitaton.hanke.ContactType.MUU
+import fi.hel.haitaton.hanke.ContactType.OMISTAJA
+import fi.hel.haitaton.hanke.ContactType.RAKENNUTTAJA
+import fi.hel.haitaton.hanke.ContactType.TOTEUTTAJA
+import fi.hel.haitaton.hanke.domain.Hanke
+import fi.hel.haitaton.hanke.domain.HankeYhteystieto
+import fi.hel.haitaton.hanke.domain.Hankealue
+import fi.hel.haitaton.hanke.geometria.GeometriatService
+import org.springframework.stereotype.Component
+
+@Component
+class HankeMapper(private val geometriatService: GeometriatService) {
+
+    fun domainFromEntity(entity: HankeEntity): Hanke =
```

Comment:
This might need to be `@Transactional` since it calls the child entities of `entity`.